projects/charger/inc/notify.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+// Module for notifying Power Distribution about charging over CAN and handling commands.
+// Requires a generic CAN module, event queue and soft timers to be running.
```

Comment:
Can you add some more detail about the intention behind this module? Like why you send messages periodically, what you expect to get in response, and what the watchdog is for.

Reply:
Done.